CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+
+If you run into an error or bug with the project:
+
+- Open an Issue at https://github.com/grafana/grafana-plugin-examples/issues
+- Include _reproduction steps_ that someone else can follow to recreate the bug or error on their own.
+- Provide project and platform versions (nodejs, npm, etc), depending on what seems relevant. If not, please be ready to provide that information if maintainers ask for it.
+
+Once it's filed:
+
+- The project team will label the issue.
+- A team member will try to reproduce the issue with your provided steps. If there are no repro steps or no obvious way to reproduce the issue, the team will ask you for those steps. Bugs will not be addressed until they are reproduced.
+- If the team is able to reproduce the issue it will be left to be [implemented by someone](#contribute-code).
+- If you or the maintainers don't respond to an issue for 30 days, the issue will be closed. If you want to come back to it, reply (once, please), and we'll reopen the existing issue. Please avoid filing new issues as extensions of one you already made.
+
+## Request a Feature
+
+If the project doesn't do something you need or want it to do:
+
+- Open an Issue at https://github.com/grafana/grafana-plugin-examples/issues
+- Provide as much context as you can about what you're running into.
+- Please try and be clear about why existing features and alternatives would not work for you.
+
+Once it's filed:
+
+- The project team will label the issue.
+- The project team will evaluate the feature request, possibly asking you more questions to understand its purpose and any relevant requirements. If the issue is closed, the team will convey their reasoning and suggest an alternative path forward.
+- If the feature request is accepted it can then be worked on by either a core team member or by anyone in the community who wants to [contribute code](#contribute-code).
+
+Note: The team is unlikely to be able to accept every single feature request that is filed. Please understand if they need to say no.
+
+## Project Setup
+
+So you wanna contribute some code! That's great! This project uses GitHub Pull Requests to manage contributions, so [read up on how to fork a GitHub project and file a PR](https://guides.github.com/activities/forking) if you've never done it before.
+
+If this seems like a lot or you aren't able to do all this setup, you might also be able to [edit the files directly](https://help.github.com/articles/editing-files-in-another-user-s-repository/) without having to do any of this setup. Yes, [even code](#contribute-code).
+
+If you want to go the usual route and run the project locally, though:
+
+- [Install Node.js](https://nodejs.org/en/download/)
+- [Fork the project](https://guides.github.com/activities/forking/#fork)
+
+Then in your terminal:
+
+- `cd path/to/your/clone/examples/<example_directory>`
+- `npm install`
+- `npm dev`
+- `mage build:backend` (if the plugin has a backend)
+- `docker compose up`
```

Comment:
```suggestion
- `docker compose up --build`
```

Reply:
What's the reasoning for forcing builds? 🤔

Reply:
It's what run server does

Reply:
Ok, let's keep it consistent as we discussed. 👍